seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -538,45 +538,34 @@
-      "title": "Interpolate a String Using Backquotes",
+      "title": "Create Strings using Template Literals",
       "description": [
-        "A new feature of ES6 or ES2015, is that it allows you to use string interpolation with back-ticks.",
-        "Consider the code below",
-        "<code>const text = 'Hello World';</code>",
-        "<code>// string interpolation</code>",
-        "<code>const divText = `</code>",
-        "<code><div></code>",
-        "<code>   ${text}</code>",
-        "<code></div></code>",
-        "<code>console.log(divText); // prints </code>",
-        "<code>//<div></code>",
-        "<code>// Hello World</code>",
-        "<code>//</div></code>",
+        "A new feature of ES6 is the <dfn>template literal</dfn>. This is a special type of string that allows you to use string interpolation features to create strings.",
+        "Consider the code below:",
+        "<blockquote>const person = {<br>  name: \"Zodiac Hasbro\",<br>  age: 56<br>};<br>// string interpolation<br>const greeting = `Hello, my name is ${person.name}!<br>I am ${person.age} years old.`;<br><br>console.log(greeting); // prints<br>// Hello, my name is Zodiac Hasbro!<br>// I am 56 years old.<br></blockquote>",
         "A lot of thing happened in there.",
-        "First, the ${variable} syntax. It's a template literal. Basically, you won't have to use concatenation with + anymore. Just drop the variable in your string, wrapped with ${ and }.",
-        "Second, we used backticks, not quotes, to wrap around the string. Notice that the string is multi-line.",
-        "In ES6, back-ticks give you more robust string creation ability.",
-        "Instructions",
-        "Use proper template literal syntax with backticks.",
-        "The expected output is each entry of result object's failure array, wrapped inside an <li> element, with class attribute text-warning.",
-        "If you have trouble finding backticks, it's the ` key, to the left of your 1; and has ~ on it."
+        "First, the <code>${variable}</code> syntax used above is a place holder. Basically, you won't have to use concatenation with the <code>+</code> operator anymore. To add variables to strings, you just drop the variable in the string and wrap it with <code>${</code> and <code>}</code>.",
+        "Second, the example uses backticks, not quotes, to wrap the string. Notice that the string is multi-line.",
+        "This new way of creating strings gives you more flexibility to create robust strings.",
+        "<hr>",
+        "Use proper template literal syntax with backticks to display each entry of the <code>result</code> object's <code>failure</code> array. Each entry should be wrapped inside an <code>li</code> element with the class attribute <code>text-warning</code>.",
+        "If you have trouble finding backticks, it's the <code>`</code> key, to the left of the <code>1</code> key and below the escape key in the upper-left corner."
```

Comment:
whoops, apologies for not thinking about that sooner! 😅 I think I may omit this altogether because there is so much variety between keyboards it would be tough to cover them all.